README.md

```diff
@@ -10,26 +10,28 @@ This Lowdefy application consists of four pages.
 
 Adding a new brand, editing an existing brand and viewing the saved brands have been split up into separate pages. This implementation is a simpler approach than that of the products. Adding a new product, editing an existing product and viewing the saved products are all done on one page. Drawers are used in order to get this right, which leads to a more complex implementation.
 
-## Running this example
+## ⚙️ Running this example
 
--  Create a MongoDB cluster and get a URI connection string:
-    - Create a free MongoDB database cluster hosted by [MongoDB Atlas](https://www.mongodb.com/try).
-    - Load the Atlas sample dataset.
-    - In the Database access section, create a database user with read access to any database (You can also specify the database as `example-crud`).
-    - In the main cluster view, click "connect", then "Connect you application". This will give a MongoDB URI connection string. Use the credentials you just created.
-    - You can read more about the [Lowdefy MongoDB connector](https://docs.lowdefy.com/MongoDB).
+- Create a MongoDB cluster and get a URI connection string:
+  - Create a free MongoDB database cluster hosted by [MongoDB Atlas](https://www.mongodb.com/try).
+  - Create a new database called `example-crud`.
```

Comment:
The mongodb driver will automatically create the database and collections when you first submit. So I don't know if this is needed. But it is nice for the reader to know about these collections

Reply:
33e3b2e12dd2f42b233293b5d6611fd66bc33178